dds_registration/models.py

```diff
@@ -192,10 +200,17 @@ def __str__(self):
 
 
 class RegistrationOption(models.Model):
+    SUPPORTED_CURRENCIES = [
+        ("USD", "US Dollar"),
+        ("CHF", "Swiss Franc"),
+        ("EUR", "Euro"),
+        ("CAD", "Canadian Dollar"),  # AKA Loonie :)
+    ]
+
     event = models.ForeignKey(Event, on_delete=models.CASCADE)
     item = models.TextField(null=False, blank=False)  # Show as an input
     price = models.FloatField(default=0, null=False)
-    add_on = models.BooleanField(default=False)
+    currency = models.TextField(choices=SUPPORTED_CURRENCIES, null=False)
```

Comment:
Should we remove `currency` field from the `Event` model, as we already have it here?

Reply:
Yes, because any event can have payment options in more than one currency. Thanks!